packages/ra-ui-materialui/src/field/TranslatableFieldsTabContent.tsx

```diff
@@ -41,34 +55,34 @@ export const TranslatableFieldsTabContent = (
         >
             {Children.map(children, field =>
                 field && isValidElement<any>(field) ? (
-                    <div key={field.props.source}>
-                        {addLabel ? (
-                            <Labeled
-                                resource={resource}
-                                label={field.props.label}
-                                source={field.props.source}
-                            >
-                                {cloneElement(field, {
-                                    ...field.props,
-                                    label: getLabel(field.props.source),
-                                    record,
-                                    source: getSource(
-                                        field.props.source,
-                                        locale
-                                    ),
-                                })}
-                            </Labeled>
-                        ) : typeof field === 'string' ? (
-                            field
-                        ) : (
-                            cloneElement(field, {
-                                ...field.props,
-                                label: getLabel(field.props.source),
-                                record,
-                                source: getSource(field.props.source, locale),
-                            })
-                        )}
-                    </div>
+                    <ResourceContextProvider
```

Comment:
I don't understand why you need this. We already are in a ResourceContext

Reply:
What about when you pass the `resource` prop?